### ISSUE TITLE
Fix documentation or / and code bug: primary_validator_schedule with node declaration

### ISSUE THREAD

**Issue by dmugtasimov** [thenewboston-developers/thenewboston-node#307]
TODO:
- <del>"primary_validator_schedule" should not present without "node" declaration: https://github.com/thenewboston-developers/thenewboston-node/wiki/thenewboston-blockchain-format#blockchain-state-example</del>
- <del>may be code should be fixed too</del>
- add blockchain state validation for this (figure out with Andrey if you need to wait until https://github.com/thenewboston-developers/thenewboston-node/issues/270 is completed)
- validate that primary validator schedule cannot be assigned if a node is not declared
- if a node is undeclared then primary validator schedule should be amended (this part will be moved out to a separate task)

This task may be completed in several small PRs on your wish

**Comment by fonar** [comment]
Hi @andbortnik, do you think that for this issue (#307) I should wait untill #270 is completed, or it is not necessary? Thanks

**Comment by dmugtasimov** [comment]
@fonar please, go ahead and do what possible. we can make a second PR once Andrey is done with his task

**Comment by dmugtasimov** [comment]
@fonar btw, this task might be partially done by me while implementing other tasks

**Comment by andbortnik** [comment]
@dmugtasimov Please review https://github.com/thenewboston-developers/thenewboston-node/pull/413

**Comment by dmugtasimov** [comment]
@andbortnik thank you merged

**Comment by andbortnik** [comment]
> if a node is undeclared then primary validator schedule should be amended (this part will be moved out to a separate task)

@dmugtasimov I'll close the task as the last part is moved out of here
